project-management/src/main/java/life/qbic/projectmanagement/application/measurement/MeasurementLookup.java

```diff
@@ -28,4 +29,10 @@ List<ProteomicsMeasurement> queryProteomicsMeasurementsBySampleIds(String filter
   List<NGSMeasurement> queryNGSMeasurementsBySampleIds(String filter,
       Collection<SampleId> sampleIds, int offset,
       int limit, List<SortOrder> sortOrders);
+
+  long countNgsMeasurementsBySampleIds(Collection<SampleId> sampleIds);
```

Comment:
Since we want to show the measurements and raw data for each domain as a seperate tab, we need to see if there are measurements registered for each domain to load the tabs in the tabsheet individually